addons/website_slides/static/src/scss/slide_course.scss

```diff
@@ -0,0 +1,536 @@
+    border-radius: 5px;
+    background-color: rgba(0,0,0,.2);
+    overflow: hidden;
+
+    .progress-gauge{
+        height: 100%;
+        width: 0;
+        background-color: $primary;
+
+        transition: all .8s ease-out;
+    }
+}
+
+.course-rating{
+    color:$primary;
+}
+.course-banner{
+    border-bottom: 1px solid $primary;
+}
+
+.content-tab{
+    .active {
+
+        border: none;
+        border-bottom: 1px solid $secondary;
+    }
+}
+
+.content-tab a:hover{
+    border:none;
+    border-bottom: 1px solid $secondary;
+}
+
+.content-container{
+    border:none;
+}
+
+.content-slide:nth-child(odd){
+    background-color: #f6f6f6;
+}
+
+.content-slide:nth-child(even){
+    background-color: #f9f9f9;
+}
+
+.content-slide-controls {
+    i:hover{
+        color: $primary !important;
+        cursor: pointer;
+    }
+}
+
+.viewer-container {
+    display: flex;
+    align-items: stretch;
+    height: 500px;
+    width: 100%;
+    background-color: #ccc;
+    justify-content: flex-start;
+}
+
+
+
+.slide-infos {
+    margin-top: 1rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: start;
+}
+
+.oe_js_player {
+
+}
+
+.slide-deactivated {
+    pointer-events: none;
+}
+
+.slide-draggable {
+    user-select: none;
+    -khtml-user-drag: element;
+    -webkit-user-drag: element;
+}
+
+.slide-hovered{
+    margin-top: 10px;
+}
+
+.section-draggable {
+    user-select: none;
+    -khtml-user-drag: element;
+    -webkit-user-drag: element;
+}
+
+.hold{
+    border: solid #ccc 4px;
+}
+
+
+.fullscreen {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 100%;
+    z-index: 1000;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+
+.sidebar-list{
+    position: fixed;
+    top: 46px;
+    bottom: 0;
+    left: 0;
+    width: 300px;
+    z-index: 2000;
+}
+
+
+
+.sidebar-list-visible {
+    left: 50px;
+}
+
+.show-list {
+    position: absolute;
+    top: 15%;
+    left: 50px;
+    font-size: 2rem;
+    z-index: 10000;
+    color: #a0a0a0;
+    border: 2px solid #ccc;
+    border-radius: 10px;
+    padding: 5px 10px;
+}
+
+.hover-background {
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 3000;
+}
+
+.hover-background:hover {
+    opacity: 1;
+}
+
+
+.show-list:hover {
+    cursor: pointer;
+}
+
+.change-small-screen {
+    font-size: 1.5rem;
+    z-index: 10000;
+    color: #a0a0a0;
+}
+
+.change-small-screen:hover {
+    cursor: pointer;
+    border-color: $primary;
+    color: $primary;
+}
+
+.sidebar-header {
+    padding: 10px 15px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 1.5rem;
+    border-bottom: 1px solid #ccc;
+    height: 80px;
+
+    a {
+        color: black;
+    }
+
+    a:hover {
+        color: $primary;
+    }
+
+    i:hover{
+        color: $primary;
+    }
+}
+
+.close-list:hover {
+    cursor: pointer;
+}
+
+.fullscreen-btn{
```

Comment:
Can't this one be included in ".fullscreen" ?

A lot of your classes are "first level", maybe more can be included in main ones for increased readability.